presto-main/src/main/java/com/facebook/presto/operator/repartition/OptimizedPartitionedOutputOperator.java

```diff
@@ -544,6 +573,7 @@ private static int getFixedWidthTypeSize(Type type)
         private final Lifespan lifespan;
         private final int capacity;
         private final int channelCount;
+        private final ArrayAllocator allocator;
```

Comment:
allocator -> arrayAllocator

Reply:
@mbasmanova I renamed it to `bufferAllocator` to be consistent with the one in PagePartitioner. Let me know if you prefer other names.